plugins/data.uribl.js

```diff
@@ -4,6 +4,7 @@
 var url       = require('url');
 var dns       = require('dns');
 var isIPv4    = require('net').isIPv4;
+var isIPv6    = require('net').isIPv6;
```

Comment:
For consistency, we've been updating these to only export `net`, and then when used, reference `net.isIPv4` and `net.isIPv6`. Otherwise, this PR looks good to me.